src/compile/legend.ts

```diff
@@ -158,6 +160,10 @@ export namespace properties {
       symbols.strokeWidth = { value: 0 };
     }
 
+    if (channel === OPACITY) {
+      delete symbols.opacity;
```

Comment:
Do you have any suggestion how should we avoid this? Since `symbols` are initialized in `applyMarkConfig`, I think it's better to do any modification in `legend.ts` instead of other places.